### PR TITLE
fix(apps/twitch-bot): update db driver to MySQL

### PR DESCRIPTION
**Pull request by durmusrasit**

### CODE REVIEW

File: apps/twitch-bot/internal/service/database/postgresql/postgresql.go [hunk 1]
```diff
@@ -19,7 +19,7 @@ type PostgreSQL struct {
 
 func NewPostgreSQL() database.Database {
 	dsn := os.Getenv("DATABASE_URL")
-	db, err := gorm.Open(postgres.Open(dsn), &gorm.Config{Logger: logger.Default.LogMode(logger.Info)})
+	db, err := gorm.Open(mysql.Open(dsn), &gorm.Config{Logger: logger.Default.LogMode(logger.Info)})
```
Comment (usirin) [inline apps/twitch-bot/internal/service/database/postgresql/postgresql.go:22]: 🤔

Comment (usirin) [reply apps/twitch-bot/internal/service/database/postgresql/postgresql.go:22]: Let's create a ticket to make sure we are not mixing postgres and mysql.

Comment (durmusrasit) [reply apps/twitch-bot/internal/service/database/postgresql/postgresql.go:22]: https://github.com/senchabot-dev/monorepo/issues/262

**Review by usirin — APPROVED** [review]
Ty for creating #262 let's hot fix this, but also make sure that we prioritize that issue as high as possible.

**Pull request merged**